wagtail/admin/static_src/wagtailadmin/scss/components/_logo.scss

```diff
@@ -23,13 +23,20 @@
 }
 
 .logo {
-    display: block;
+    display: flex;
```

Comment:
In my testing this makes the "Dashboard" text stick out of the nav.

![wip-highlight-over-link](https://user-images.githubusercontent.com/877585/47614053-88402200-daaa-11e8-949f-68891f1ab662.png)